doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1549 @@
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script` script:
+
+  * Added semantic version support for devices that use the :ref:`SUIT DFU <ug_nrf54h20_suit_dfu>`.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added:
+
+  * :ref:`Compressed firmware update support <mcuboot_image_compression>`.
+  * Support for SHA-512.
+  * Support for ED25519 'PURE' signature (``CONFIG_BOOT_SIGNATURE_TYPE_PURE`` in MCUboot).
+  * Integration of PSA crypto API as cryptographic backend for integration with cryptographic acceleration on the nRF54L15 SoC (``CONFIG_BOOT_USE_PSA_CRYPTO`` in MCUboot).
+  * Support for signature verification using hardware cryptography (Cracken) on the nRF54L15 SoC.
+    This is supported either for the ED25519 'PURE' signature or the ED25519 signature.
+  * Support for storing signature verification keys in hardware KMU on the nRF54L15 SoC.
+    See Kconfig option :kconfig:option:`CONFIG_BOOT_SIGNATURE_USING_KMU` in MCUboot for details.
+  * Integration of nRF54L15 SoC's HW cryptography (``SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519``), KMU sysbuild configuration (``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU``), and signing script (``SB_CONFIG_BOOT_SIGNATURE_TYPE_PURE``) for building and signing the application in a project.
+
+* Updated the immutable bootloader to now be protected using the :ref:`fprotect_readme` library on the nRF54L15 SoC (``SB_CONFIG_MCUBOOT_FPROTECT_ALLOW_COMBINED_REGIONS`` and the :kconfig:option:`CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS` Kconfig option).
+
+* Fixed:
+
+  * An issue with wrong network core address checking in MCUboot for nRF5340 network core firmware updates.
+  * An issue with MCUboot updates in MCUboot when a swap mode was selected, which could lead to possible data corruption by implementing additional NSIB-specific (overwrite-only) upgrade mode for sysbuild.
+  * Multiple issues with checking firmware update slot selection in MCUboot for sysbuild.
+  * An issue with MCUboot cleanup unused slots feature, which could lead to possible empty firmware update of the main image and bricking the device for sysbuild.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``ea02b93eea35afef32ebb31f49f8e79932e7deee``, with some |NCS| specific additions.
```

Comment:
This SHA seems wrong, looks like a copy-paste from the previous release. The actual merge-base for Zephyr is now https://github.com/zephyrproject-rtos/zephyr/commit/beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee, need to update this